source/getting-started/index.md

```diff
@@ -1,52 +1,55 @@
 Getting started with Ember.js is easy. Ember.js projects are created and managed
-through our [command line build tool ember-cli](http://www.ember-cli.com/). The command line tool brings:
-
-* Modern application asset management (including combining, minifying, and versioning).
-* Built-in generators to help you create components, routes, and more.
-* A conventional project layout so approaching other developers' Ember.js applications is easy.
-* Official [JavaScript modules](http://jsmodules.io/) to keep your project organized.
-* A complete testing framework.
-* Access to a growing ecosystem of [Ember Addons](http://www.emberaddons.com/).
-
+through our command line build tool [Ember CLI](http://www.ember-cli.com/).
```

Comment:
It has [mostly](https://github.com/emberjs/guides/pulls?q=is%3Apr+author%3Ajoostdevries+is%3Aclosed) [happened](https://github.com/emberjs/guides/pull/528) and is [wrapping up shortly](https://github.com/emberjs/guides/pulls/joostdevries).